### PR TITLE
Replace String.Format with string interpolation

### PR DESCRIPTION
**Pull request by mikkelbu**
Also added language identifiers to some code blocks in the wcf folder.

Relates to #9002 and #2192

### CODE REVIEW

File: docs/framework/data/adonet/sql/windows-applications-using-callbacks.md [hunk 1]
```diff
@@ -352,7 +349,7 @@ private void HandleCallback(IAsyncResult result)
         // You can create the delegate instance as you   
         // invoke it, like this:  
         this.Invoke(new DisplayInfoDelegate(DisplayStatus),  
-        String.Format("Ready(last error: {0}", ex.Message));  
+            $"Ready(last error: {ex.Message}");
```
Comment (pkulikov) [inline docs/framework/data/adonet/sql/windows-applications-using-callbacks.md]: @mikkelbu nit not related to your changes: it should be a white space after "Ready"

File: docs/framework/data/adonet/updating-data-sources-with-dataadapters.md [hunk 2]
```diff
@@ -290,7 +290,7 @@ class Program {
       String primaryCols = String.Join(",", primaryColumns.Select(col => col.ColumnName));  
       String resetCols = String.Join(",", resetColumns.Select(col => "Max(" + col.ColumnName + ") as " + col.ColumnName));
```
Comment (pkulikov) [inline docs/framework/data/adonet/updating-data-sources-with-dataadapters.md]: not the `string.Format` call, but still this is also the candidate for string interpolation?
```csharp
col => $"Max({col.ColumnName}) as {col.ColumnName}"
```

File: docs/framework/data/adonet/sql/windows-applications-using-callbacks.md (hunk 1 above)
Comment (mikkelbu) [reply docs/framework/data/adonet/sql/windows-applications-using-callbacks.md]: @pkulikov I've added a space here, and also in a similar case in the same file.

File: docs/framework/data/adonet/updating-data-sources-with-dataadapters.md (hunk 2 above)
Comment (mikkelbu) [reply docs/framework/data/adonet/updating-data-sources-with-dataadapters.md]: I've replaced the string concatenation with string interpolation.

Comment (pkulikov) [reply docs/framework/data/adonet/updating-data-sources-with-dataadapters.md]: Thanks!

**Review by rpetrusha — APPROVED** [review]
These changes look good, @mikkelbu. Thanks for making them, including all of the improvements unrelated to string interpolation. I'll merge your PR now.

**Pull request merged**